mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -827,30 +821,29 @@ public void testIcebergAndHmsTableProperties() throws Exception {
             .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
 
     if (Catalogs.hiveCatalog(shell.getHiveConf(), tableProperties)) {
-      // 2 newly-added properties + previous_metadata_location prop + explicit Parquet compression
-      Assert.assertEquals(17, hmsParams.size());
-      Assert.assertEquals("true", hmsParams.get("new_prop_1"));
-      Assert.assertEquals("false", hmsParams.get("new_prop_2"));
-      Assert.assertEquals("new_val", hmsParams.get("custom_property"));
+      assertThat(hmsParams).hasSize(17);
+      assertThat(hmsParams).containsEntry("new_prop_1", "true");
+      assertThat(hmsParams).containsEntry("new_prop_2", "false");
+      assertThat(hmsParams).containsEntry("custom_property", "new_val");
       String prevSnapshot = getCurrentSnapshotForHiveCatalogTable(icebergTable);
       icebergTable.refresh();
       String newSnapshot = getCurrentSnapshotForHiveCatalogTable(icebergTable);
-      Assert.assertEquals(
-          hmsParams.get(BaseMetastoreTableOperations.PREVIOUS_METADATA_LOCATION_PROP),
-          prevSnapshot);
-      Assert.assertEquals(
-          hmsParams.get(BaseMetastoreTableOperations.METADATA_LOCATION_PROP), newSnapshot);
+      assertThat(hmsParams)
+          .containsEntry(
+              BaseMetastoreTableOperations.PREVIOUS_METADATA_LOCATION_PROP, prevSnapshot);
+      assertThat(hmsParams)
+          .containsEntry(BaseMetastoreTableOperations.METADATA_LOCATION_PROP, newSnapshot);
     } else {
-      Assert.assertEquals(8, hmsParams.size());
+      assertThat(hmsParams).hasSize(8);
     }
 
     // Remove some Iceberg props and see if they're removed from HMS table props as well
     if (Catalogs.hiveCatalog(shell.getHiveConf(), tableProperties)) {
       icebergTable.updateProperties().remove("custom_property").remove("new_prop_1").commit();
       hmsParams = shell.metastore().getTable("default", "customers").getParameters();
-      Assert.assertFalse(hmsParams.containsKey("custom_property"));
-      Assert.assertFalse(hmsParams.containsKey("new_prop_1"));
-      Assert.assertTrue(hmsParams.containsKey("new_prop_2"));
+      assertThat(hmsParams).doesNotContainKey("custom_property");
```

Comment:
these can be chained together: 
```
assertThat(hmsParams)
    .doesNotContainKey("custom_property") 
    .doesNotContainKey("new_prop_1")
    .containsKey("new_prop_2");
```

Reply:
can you also please chain together the other assertions against `hmsParams`

Reply:
oh, thanks, you teach me something new! I'll fix them.